packages/perseus/src/dependencies.ts

```diff
@@ -22,6 +22,10 @@ export const getDependencies = (): PerseusDependencies => {
     );
 };
 
+export type DependencyProps = Partial<
```

Comment:
I love that we're adding this.  Could I ask that you circle back on a separate PR and make sure other sites that used this `Partial<React.ContextType...` stuff be replaced with this new type?